src/render/camera/camera.rs

```diff
@@ -128,14 +138,22 @@ impl CalculatedFrustrum {
 
             let mut positive = aabb.min;
             const ZERO: f32x4 = f32x4::from_array([0.0, 0.0, 0.0, 0.0]);
-            let cmp = f32x4::from_array([n.x, n.y, n.z, 0.0]).simd_ge(ZERO).to_array();
+            let cmp = f32x4::from_array([n.x, n.y, n.z, 0.0])
+                .simd_ge(ZERO)
+                .to_array();
 
-            if cmp[0] { positive.x = aabb.max.x }
-            if cmp[1] { positive.y = aabb.max.y }
-            if cmp[2] { positive.z = aabb.max.z }
+            if cmp[0] {
+                positive.x = aabb.max.x
+            }
+            if cmp[1] {
+                positive.y = aabb.max.y
+            }
+            if cmp[2] {
+                positive.z = aabb.max.z
+            }
```

Comment:
I think this is unnecessary

Reply:
yeah, I have no idea why rustfmt did this.